example/swagger-files/parameter-styles-query.json

```diff
@@ -0,0 +1,247 @@
+                        "type": "array",
+                        "items": {
+                            "type": "string"
+                        }
+                    }
+                }, {
+                    "name": "object",
+                    "in": "query",
+                    "description": "spaceDelimited style object. Note: This does not currently work. See associated test.",
+                    "required": true,
+                    "style": "spaceDelimited",
+                    "explode": false,
+                    "schema": {
+                        "type": "object",
+                        "properties": {
+                            "name": {
+                                "type": "string"
+                            },
+                            "description": {
+                                "type": "string"
+                            }
+                        }
+                    }
+                }],
+                "responses": {
+                    "200": {
+                        "description": "Successful response"
+                    }
+                }
+            }
+        },
+        "/nonExplodedPipeDelimitedQueries": {
+            "get": {
+                "summary": "Demonstrate all non-exploded pipeDelimited query styles",
+                "operationId": "demonstrateQueries",
+                "parameters": [{
+                    "name": "array",
+                    "in": "query",
+                    "description": "pipeDelimited style array",
+                    "required": true,
+                    "style": "pipeDelimited",
+                    "explode": false,
+                    "schema": {
+                        "type": "array",
+                        "items": {
+                            "type": "string"
+                        }
+                    }
+                }, {
+                    "name": "object",
+                    "in": "query",
+                    "description": "pipeDelimited style object. Note: This does not currently work. See associated test.",
+                    "required": true,
+                    "style": "pipeDelimited",
+                    "explode": false,
+                    "schema": {
+                        "type": "object",
+                        "properties": {
+                            "name": {
+                                "type": "string"
+                            },
+                            "description": {
+                                "type": "string"
+                            }
+                        }
+                    }
+                }],
+                "responses": {
+                    "200": {
+                        "description": "Successful response"
+                    }
+                }
+            }
+        },
+        "/explodedDeepObjectQueries": {
+            "get": {
+                "summary": "Demonstrate all non-exploded deep object query styles",
```

Comment:
This one's throwing some errors right now on a fresh page load.

![Screen Shot 2021-01-06 at 10 44 35 AM](https://user-images.githubusercontent.com/33762/103808028-342b1380-500c-11eb-9d20-59ff260e01f7.png)

Reply:
fixed locally